src/test/scala/io/iohk/ethereum/jsonrpc/EthServiceSpec.scala

```diff
@@ -205,13 +210,21 @@ class EthServiceSpec extends FlatSpec with Matchers with ScalaFutures {
     )
   }
 
+  it should "return requested work" in new TestSetup {
+    (blockGenerator.generateBlockForMining _).expects().returning(block)
+
+    val response = ethService.getWork(GetWorkRequest())
+
+    response.futureValue shouldEqual GetWorkResponse(powHash, seedHash, target)
+  }
+
```

Comment:
I guess that's ok if you plan to do it in the next PR, but I would normally recommend adding and empty ignored/pending test and a TODO comment.